src/the-2021-scheduler.md

```diff
@@ -14,12 +14,7 @@ Client support:
 - Anki: 2.1.45+
 - AnkiMobile: 2.0.75+
 - AnkiWeb: yes
-- AnkiDroid: available in the alpha builds, after enabling the new schema
-in the advanced prefs.
-
-It will likely be some time until a stable AnkiDroid release supports the v3
-scheduler directly. In the mean time, it will continue to show cards using the
-v2 scheduler, regardless of the setting you have chosen.
+- AnkiDroid: 2.16.2+
```

Comment:
```suggestion
- AnkiDroid: 2.16.2+ includes support for the v3 scheduler when the experimental "new backend" option is enabled in the advanced preferences. Support for custom scheduling (for things like FSRS) is not yet implemented.
```

Reply:
You're right, I didn't realise that with just the change log.